### PR TITLE
Issue #72 Remove Force Unwrapping

### PR DESCRIPTION
**Pull request by annie-gupta**
Removed Force Unwrapping throughout the App

### CODE REVIEW

**Review by DragonCat4012 — COMMENTED** [review]
Otherwise the pr targets master, we swicthted to stable as our "main" branch. pls change our pr to target this branch

File: Calq/Screens/ExamScreen/ExamScreen.swift [hunk 1]
```diff
@@ -103,16 +103,20 @@ struct ExamView: View {
         .padding(.horizontal, 10)
         .padding(.vertical, 5)
         .onAppear {
-            subject = getExam(type)
-            sliderValue = (subject != nil) ? Float(Int(subject!.exampoints)) : 0
+            sliderValue = 0
+            if let subject = getExam(type) {
+                sliderValue = Float(Int(subject.exampoints))
+            }
```
Comment (A-K-O-R-A) [inline Calq/Screens/ExamScreen/ExamScreen.swift:109]: This is a big improvement to the previous to the previous code and a lot more readable. So thank you :+1:
Wouldn't it also be useful to add a debug log to catch instances of the `getExam()` returning `nil` as this should not happen?

Comment (DragonCat4012) [reply Calq/Screens/ExamScreen/ExamScreen.swift:109]: yes sounds usefull 👍

**Comment by annie-gupta** [comment]
@DragonCat4012 : Have changed the base branch to stable

**Comment by DragonCat4012** [comment]
culd it be you merged the stable branch into it? bc th last edited and created things are now in there, which are only on master and not stable. the whole point of the stable branch was to not include these core data changes

**Comment by annie-gupta** [comment]
> culd it be you merged the stable branch into it? bc th last edited and created things are now in there, which are only on master and not stable. the whole point of the stable branch was to not include these core data changes

@DragonCat4012 Have merged to the updated stable branch omitting changes from main. Please validate.

**Review by DragonCat4012 — APPROVED** [review]
looks good 👍

**Pull request merged**